app/assets/stylesheets/layouts/_header.scss

```diff
@@ -37,16 +29,34 @@
       margin: 0;
     }
   }
-}
 
-.c-header {
+  .icon-menu {
```

Comment:
We should create (if it doesn't exist yet) an `_icons.scss` file. You can set the sizes and colors inside of this file.